server/services/utils/getServicesForUser.test.ts

```diff
@@ -768,10 +770,63 @@ describe('getServicesForUser', () => {
       [[Role.CreateAnEMOrder], [{ app: ServiceName.CEMO, activeAgencies: ['ANOTHER'] }], 'LEI', false],
       [[Role.CreateAnEMOrder], [{ app: ServiceName.CEMO, activeAgencies: ['LEI'] }], 'LEI', true],
     ])('user with roles %s, can see: %s', (roles, activeServices, activeCaseLoadId, visible) => {
-      const output = getServicesForUser(roles, false, activeCaseLoadId, 12345, [], activeServices)
+      const output = getServicesForUser(roles, false, { policies: [] }, activeCaseLoadId, 12345, [], activeServices)
       expect(
         output.some(service => service.heading === 'Apply, change or end an Electronic Monitoring Order (EMO)'),
       ).toEqual(visible)
     })
   })
+
```

Comment:
I don't think there's any tests for showing that the legacy UI is disabled when the new is enabled